enterprise/frontend/src/embedding-sdk/css/index.module.css

```diff
@@ -0,0 +1,3 @@
+svg {
```

Comment:
should we necessarily have a global selector here? it could leak into the client app. Maybe we can put it under a `#metabase` ID or class or something